doc/user/content/sql/functions/windowed-aggregates.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Windowed Aggregate Functions"
```

Comment:
Nitpick: Do we want to call them that, even though they are not aggregate functions, and just functions to better support windowed aggregations?

Reply:
Very open to suggestions here.